wsgi/playdoh.wsgi

```diff
@@ -20,25 +20,22 @@ if newrelic:
         newrelic = False
 
 
-# NOTE: you can also set DJANGO_SETTINGS_MODULE in your environment to override
-# the default value in manage.py
+# Set the settings module explicitly if we don't have one in the
+# environment already.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
 
 # Add the app dir to the python path so we can import manage.
 wsgidir = os.path.dirname(__file__)
 site.addsitedir(os.path.abspath(os.path.join(wsgidir, '../')))
 
-# Explicitly set these so that fjord.manage_utils does the right
-# thing in production.
-os.environ['USING_VENDOR'] = '1'
```

Comment:
It doesn't need to change in production because this no longer imports `manage.py` for side-effects. Instead it does the things it needs to do explicitly. i.e. `setup_environ()` and `monkeypatch()`.